web/src/Sidebar.js

```diff
@@ -1,9 +1,13 @@
-import React, { Component } from 'react';
+import React, { PureComponent } from 'react';
 import './Sidebar.scss';
 
-class Sidebar extends Component {
+class Sidebar extends PureComponent {
   render() {
-    return (<div className="Sidebar">I'm a sidebar!</div>)
+    let classes = ['Sidebar']
```

Comment:
semicolon?

Reply:
why?

Reply:
I come from the "semicolons everywhere" school of thought to avoid ambiguity but I don't feel strongly about it.